housekeeping-vacuum-tool/src/main/assembly/housekeeping-tool-package.xml

```diff
@@ -0,0 +1,44 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+  <id>bin</id>
+  <formats>
+    <format>tgz</format>
+  </formats>
+  <baseDirectory>housekeeping-vacuum-tool-${project.version}</baseDirectory>
+  <dependencySets>
+    <dependencySet>
+      <outputDirectory>lib</outputDirectory>
+      <useProjectArtifact>true</useProjectArtifact>
```

Comment:
I don't think you want the project artifact, you want the "-all" jar that the shade plugin is producing. Have you actually tested this end to end using the .tgz version? I'd be surprised if it works as currently set up.